databay/link.py

```diff
@@ -78,8 +75,11 @@ def __init__(self,
         self._outlets = []
         self.add_inlets(inlets)
         self.add_outlets(outlets)
-        self._interval = interval
-        self._transfer_number = -1
+        if isinstance(interval, (int, float)):
+            self._interval = datetime.timedelta(seconds=interval)
+        else:
+            self._interval = interval
+        self._count = -1
```

Comment:
here we got another merge conflict with #13 - it should have been updated to `_transfer_number`. Conflicts can be a pain eh! Fixing it too, just letting you know 👍